packages/hardhat-chai-matchers/src/internal/changeTokenBalance.ts

```diff
@@ -57,8 +67,11 @@ export function supportChangeTokenBalance(Assertion: Chai.AssertionStatic) {
       balanceChanges: EthersT.BigNumberish[]
     ) {
       const ethers = require("ethers") as typeof EthersT;
-      let subject = this._obj;
 
+      // see buildAssert's jsdoc
```

Comment:
I'd like to see a one-line explanation here, something like `capture negated flag before async code executes. see buildAssert's jsdoc.`